cpp/src/routing/local_search/sliding_tsp.cu

```diff
@@ -275,7 +275,7 @@ __global__ void execute_sliding_moves_tsp(
   s_route.copy_from(route);
   __syncthreads();
 
-  s_route.copy_to_tsp_route(sol.problem.order_info.depot_included);
```

Comment:
Why is it handled differently for TSP?

Reply:
Since we have move recycling this representation provides ~30% faster move execution. We only update pred/succ pointers and copying to route_t at the end rather than shifting route_t to eject/insert nodes for each non overlapping move.